### DISCUSSION TITLE
Does Chronos-Bolt still perform tokenization

### DISCUSSION THREAD

**Discussion by shenxinyuan-hash** [amazon-science/chronos-forecasting#248] [Q&A] [answered]
Hello, may I inquire whether Chronos-Bolt still adheres to the tokenization methodology employed by Chronos-T5?

**Comment by lostella** [comment, marked as answer]
@shenxinyuan-hash no, the Chronos-Bolt models embed non-overlapping patches of the context data, and output direct multistep quantile estimates. Check out this thread for more details https://github.com/amazon-science/chronos-forecasting/discussions/240

**Reply by shenxinyuan-hash** [reply]
Thank you very much!
